src/coreclr/jit/compiler.h

```diff
@@ -8925,8 +8925,9 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 public:
     enum UnrollKind
     {
-        Memset, // Initializing memory with some value
-        Memcpy  // Copying memory from src to dst
+        Memset,
+        Memcpy,
+        Memmove
```

Comment:
it is used in `LowerCallMemmove`. Currently it's not handled in `getThreshould` inside and treated just like memcpy, but it allows future changes for memcpy only since memmove has a hard limit to grow.